pyproj/types.py

```diff
@@ -0,0 +1,18 @@
+from collections import namedtuple
+
+GeodIntermediateReturn = namedtuple(
```

Comment:
Mind moving this to _geod.pyx since it is no longer a type?

Reply:
> Mind moving this to _geod.pyx since it is no longer a type?

It is used in `geod.py` as well.

Reply:
> It is used in geod.py as well.

Should be able to import it from `_geod` without too much trouble.

Reply:
OK, done.